domaingraph_test.go

```diff
@@ -158,3 +134,54 @@ func TestClientReloadDomainGraph(t *testing.T) {
 		t.Fatalf("failed reloading domain graph: %v", err)
 	}
 }
+
+func TestClientReloadDomainGraphTimestamp(t *testing.T) {
+	expectedBodyString := `{
+		"entities": [],
+		"relationships": [],
+		"historyTimestamp": "2020-06-15T16:26:10Z"
+	}`
+
+	var fun = domainGraphHandlerFunc(expectedBodyString, t)
+
+	client, server := setup(t, fun)
```

Comment:
Nit: you could refactor `domainGraphHandlerFunc` to something like `setupDomainGraphServer` which returns a client and server directly, since that's what you need in both tests anyway.